pydoc/setup/LINUX_INSTALL.md

```diff
@@ -18,7 +18,7 @@ cd chapel-1.29.0/
 export CHPL_HOME=$PWD
 
 # Add chpl to PATH
-source $CHPL_HOME/util/setchplenv.bash
+source $CHPL_HOME/server_util/setchplenv.bash
```

Comment:
`server_util` does not exist here. This is referring to a directory in `$CHPL_HOME` and should remain `util`.

Reply:
Fixed